### PR TITLE
[release-4.8] Bug 1977342: Fix obfuscation translation table secret 4.8

### PR DESCRIPTION
**Pull request by Sergey1011010**
Backport [CCXDEV-4481](https://issues.redhat.com/browse/CCXDEV-4481) to 4.8

## Categories

- [X] Bugfix
- [ ] Enhancement
- [ ] Backporting
- [ ] Others (CI, Infrastructure, Documentation)

## Sample Archive

No changes

## Documentation

No changes

## Unit Tests

- `pkg/anonymization/anonymizer_test.go`

## Privacy

No new collected data

## Changelog

No

## Breaking Changes

No

## References

https://issues.redhat.com/browse/CCXDEV-5297
https://bugzilla.redhat.com/show_bug.cgi?id=???
https://access.redhat.com/solutions/???

### CODE REVIEW

**Comment by tremes** [comment]
This looks like a mix with https://github.com/openshift/insights-operator/pull/462 (or not? :))). I don't know if this is the intention, but we should be consistent (especially in the backports) so please update or rebase against release-4.8.

File: manifests/03-clusterrole.yaml [hunk 1]
```diff
@@ -348,6 +323,84 @@ roleRef:
...
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/single-node-developer: "true"
+subjects:
+  - kind: ServiceAccount
+    name: operator
+    namespace: openshift-insights
+roleRef:
+  kind: Role
+  name: insights-operator-etc-pki-entitlement
```
Comment (tremes) [inline manifests/03-clusterrole.yaml:406]: The `insights-operator-etc-pki-entitlement` role and the corresponding rolebinding shouldn't be here. Pls remove.

Comment (Sergey1011010) [reply manifests/03-clusterrole.yaml]: Fixed

**Comment by tremes** [comment]
Reviewed & hopefully verified in my cluster. Thanks!!
/lgtm

**Pull request merged**